recipes/example/meta.yaml

```diff
@@ -1,70 +0,0 @@
-# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```

Comment:
Yes, it is really necessary.  Removing the example breaks CI for all of staged recipes.